### PR TITLE
Smoke test

### LINKED ISSUE #19: Smoke test that rendering works for the default configuration

### PR DESCRIPTION
**Pull request by Korla**
closes #19

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+configure({ adapter: new Adapter() });
```
Comment (pardahlman) [inline src/setupTests.js]: what is the purpose of this file? It doesn't look like it is used? 🤔

Comment (Korla) [reply src/setupTests.js]: It is [the configuration for enzyme](https://create-react-app.dev/docs/running-tests/#option-1-shallow-rendering). Apparently they have adapters for each react version. `setupTests.js` is [picked up by react-scripts](https://create-react-app.dev/docs/running-tests/#initializing-test-environment).

Comment (Korla) [reply src/setupTests.js]: I forgot to write this in the PR comments.

**Comment by pardahlman** [comment]
Rebase, resolve and we're all good 👍

**Pull request merged**